src/bin/dual-iir.rs

```diff
@@ -82,7 +82,7 @@ pub struct Settings {
     ///
     /// # Value
     /// Any of the variants of [Gain] enclosed in double quotes.
-    #[miniconf(defer)]
+    #[tree()]
```

Comment:
Can we update it so that we can just annotate it with `#[tree]`? The parenthesis seem odd

Reply:
https://github.com/quartiq/miniconf/pull/176/commits/fa6ff4bbc329c99fccfd1cc6cf4c776450959b89